### PR TITLE
done cart page

### PR DESCRIPTION
**Pull request by srishtigautam21**

### CODE REVIEW

**Review by purohitdheeraj — APPROVED** [review]
Good to merge

File: Pages/CartPage/cart.css [hunk 1]
```diff
@@ -0,0 +1,84 @@
+.cart-page-container{
+    display: flex;
+    /* column-gap: 4rem; */
+    justify-content:space-around;
```
Comment (Shubhamkr2610) [inline Pages/CartPage/cart.css:4]: remove unused code

**Review by itsmebipul00 — APPROVED** [review]

**Review by tonymike99 — APPROVED** [review]

**Review by archit7448 — APPROVED** [review]
looks good to merge

File: Pages/CartPage/cart.css (hunk 1 above)
Comment (srishtigautam21) [reply Pages/CartPage/cart.css:4]: ok will do it

File: Pages/CartPage/cart.css [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+    flex:0 0 auto;
+}
+.quantity{
+    display: flex;
+    margin: 2rem 0;
+    column-gap: 1.5rem;
+}
+.plus-minus-button{
+    display:flex;
+}
```
Comment (zoef-yt) [inline Pages/CartPage/cart.css:36]: make a class of display flex in utils and use it wherever needed

File: common.css [hunk 3]
```diff
@@ -2,6 +2,7 @@
 @import url('/Pages/ProductPage/product.css');
 @import url('/Pages/LoginPage/login.css');
 @import url('/Pages/SignupPage/signup.css');
+@import url('/Pages/CartPage/cart.css');
 
 .main-wrapper{
     display: flex;
```
Comment (zoef-yt) [inline common.css:8]: that class can be used directly in this element.

**Review by zoef-yt — APPROVED** [review]
The above comment is just a suggestion you can add. Good code. Good to merge.

File: Pages/CartPage/cart.css (hunk 2 above)
Comment (srishtigautam21) [reply Pages/CartPage/cart.css:36]: ok will look into it

File: common.css (hunk 3 above)
Comment (srishtigautam21) [reply common.css:8]: ok thanks for the suggestions i will look into them

**Pull request merged**